src/js/_enqueues/wp/api-request.js

```diff
@@ -55,18 +55,21 @@
 
 		// If ?_wpnonce=... is present, no need to add a nonce header.
 		addNonceHeader = ! ( options.data && options.data._wpnonce );
+		addAcceptHeader = true;
 
 		headers = options.headers || {};
 
-		// If an 'X-WP-Nonce' header (or any case-insensitive variation
-		// thereof) was specified, no need to add a nonce header.
-		if ( addNonceHeader ) {
-			for ( headerName in headers ) {
-				if ( headers.hasOwnProperty( headerName ) ) {
-					if ( headerName.toLowerCase() === 'x-wp-nonce' ) {
-						addNonceHeader = false;
-						break;
-					}
+		for ( headerName in headers ) {
+			if ( headers.hasOwnProperty( headerName ) ) {
+				// If an 'X-WP-Nonce' header (or any case-insensitive variation
```

Comment:
This section would read a little cleaner as a switch statement:
```js
switch( headerName.toLowerCase() ) {
	case 'x-wp-nonce':
		addNonceHeader = false;
		break;
	case 'accept':
		addAcceptHeader = false;
		break;
}

Reply:
Fixed in a316b9fff86b4afc34753318de558fe46c399d14.